.github/renovate.json

```diff
@@ -17,6 +17,59 @@
     {
       "matchPackageNames": ["node"],
       "allowedVersions": "/20.[0-9]+.[0-9]+(.[0-9]+)?$/"
+    },
+    {
+      "matchPackagePatterns": ["(org.eclipse.jetty:jetty-maven-plugin|org.jenkins-ci:winstone)"],
+      "description": "Should be upgraded in lockstep in order to keep their corresponding Jetty versions aligned, could be grouped but releases are likely separated by a bit of time",
+      "matchManagers": ["maven"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["jakarta.servlet:jakarta.servlet-api"],
+      "description": "Provided by Jetty and should be aligned with the version provided by the version of Jetty we deliver. See: https://github.com/jenkinsci/jenkins/pull/5211",
+      "matchManagers": ["maven"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api"],
+      "description": "Contains incompatible API changes and needs compatibility work",
+      "matchManagers": ["maven"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["org.codehaus.groovy:groovy-all"],
+      "description": "Needs significant testing. See: https://github.com/jenkinsci/jenkins/pull/5112#issuecomment-744429487 and https://github.com/jenkinsci/jenkins/pull/5116#issuecomment-744526638",
+      "matchManagers": ["maven"],
+      "allowedVersions": "<2.5.0"
+    },
+    {
+      "matchPackagePatterns": ["org.fusesource.jansi:jansi"],
+      "description": "Consumed by Groovy and should be updated in lockstep with Groovy. See: https://github.com/jenkinsci/jenkins/pull/5184",
+      "matchManagers": ["maven"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["org.jfree:jfreechart"],
+      "description": "Contains incompatible API changes and needs compatibility work. See: https://github.com/jenkinsci/jenkins/pull/4224",
+      "matchManagers": ["maven"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["org.springframework"],
+      "description": "Starting with 6.x, Spring requires Java 17 at a minimum.",
+      "matchManagers": ["maven"],
+      "allowedVersions": "<6.0.0"
+    },
+    {
+      "matchPackagePatterns": ["com.google.inject:guice-bom"],
+      "description": "Starting with 7.x, Guice switches from javax.* to jakarta.* bindings. See https://github.com/google/guice/wiki/Guice700",
+      "matchManagers": ["maven"],
+      "allowedVersions": "<7.0.0"
+    },
+    {
+      "matchFileNames": ["test/pom.xml", ".mvn/maven.config"],
+      "matchPackageNames": ["org.jenkins-ci.main:remoting"],
+      "enabled": false
```

Comment:
this rule doesn't match `pom.xml`, so renovate still suggests an update for `org.jenkins-ci.main:remoting` inside the root `pom.xml`

Reply:
yeah there's one `org.jenkins-ci.main:remoting` that we want updated, the `remoting.version` one which is the latest version.

We don't want the 'old version for testing', and the 'minimum supported version' updated. Which is all working with this setup now by moving it to `.mvn/maven.config`